spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/OpenTelemetryConfigurationsTracerConfigurationTests.java

```diff
@@ -59,19 +59,18 @@ void shouldNotSupplyTracerIfOpenTelemetryIsMissing() {
 
 	@Test
 	void shouldBackOffOnCustomBeans() {
-		this.contextRunner.withUserConfiguration(OpenTelemetryConfiguration.class, CustomConfiguration.class)
-				.run((context) -> {
-					assertThat(context).hasBean("customTracer");
-					assertThat(context).hasSingleBean(Tracer.class);
-				});
+		this.contextRunner.withUserConfiguration(CustomConfiguration.class).run((context) -> {
```

Comment:
I think we need `OpenTelemetryConfiguration` here so that `@ConditionalOnBean(OpenTelemetry.class)` on `OpenTelemetryConfigurations.TracerConfiguration.otelTracer(OpenTelemetry)` is satisfied and the test is then checking the `@ConditionalOnMissingBean`.

Reply:
@wilkinsona Oops, I missed it. I reverted this change.